.github/workflows/ci.yml

```
@@ -55,6 +56,11 @@ jobs:
     - name: Run black
       run: |
         black --check --diff --exclude="contrib" .
+    - name: Run ESLint
+      run: eslint
+    - name: Run djlint
+      run: |
+        djlint basicswap/templates/ --lint
```

```suggestion
    - name: Run ESLint
      run: eslint || true
    - name: Run djlint
      run: |
        djlint basicswap/templates/ --lint || true
```

can make these exit with code 0 to allow the ci to complete green, but still print the info